afterpay/build.gradle.kts

```diff
@@ -0,0 +1,75 @@
+plugins {
+    alias(libs.plugins.android.library)
+    alias(libs.plugins.kotlin.android)
+    alias(libs.plugins.kotlin.parcelize)
+    alias(libs.plugins.vanniktech.maven.publish)
+    alias(libs.plugins.kotlin.serialization)
+}
+
+android {
+    compileSdk = libs.versions.compileSdkVersion.get().toInt()
+    buildToolsVersion = libs.versions.buildTools.get()
+
+    defaultConfig {
+        minSdk = libs.versions.minSdk.get().toInt()
+        targetSdk = libs.versions.compileSdkVersion.get().toInt()
+
+        val VERSION_NAME: String by project
+        buildConfigField("String", "AfterpayLibraryVersion", "\"$VERSION_NAME\"")
+
+        consumerProguardFiles("consumer-rules.pro")
+    }
+
+    buildTypes {
+        create("staging") {
+            initWith(getByName("debug"))
+        }
+    }
+
+    sourceSets {
+        val main by getting
+        main.java.srcDirs("src/main/kotlin")
```

Comment:
Doesn't the Kotlin plugin pick these sourcesets up automatically?

Reply:
Probably.  Looks like everything still compiles w/ these removed.